docs/en/week08/08-3.md

```diff
@@ -0,0 +1,198 @@
+---
+lang-ref: ch.08-3
+title: Generative Models - Autoencoders
+lecturer: Alfredo Canziani
+authors: Nandhitha Raghuram, Xinyi Zhao
+date: 25 March 2021
+---
+
+### Recap: Predictive Latent Variable Model
+As we know latent variables are useful in providing additional information that help in predicting multiple values of the target $y$, assiociated with the input $x$. We have seen earlier, in order to predict the target variable $y$ from latent variable $z$, the energy function is given as,
+$$E(y,z) = [y_1 - g_1(z)]^2 + [y_2 - g_2(z)]^2, y \in Y$$
+where $g = [g_{1}, g_{2}]^{T}$, $\mathbb{R} \rightarrow \mathbb{R}^{2}$ and $g(z) = [w_{1}cos(z), w_{2}sin(z)]$.
```

Comment:
The transposition symbol is `^\top`.
`\cos` and `\sin` need a `\`.

Reply:
fixed